examples/basic-with-apply/src/playground.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { FocalPoint } from '@lemoncode/react-image-focal-point';
+import classes from './playground.module.css';
+
+interface Props {
+  src: string;
+  focalPoint?: FocalPoint;
+  children: React.ReactNode;
+}
+
+export const Playground: React.FC<Props> = props => {
+  const { src, children } = props;
+  const [focalPoint, setFocalPoint] = React.useState<FocalPoint>();
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (props.focalPoint) {
+      setFocalPoint(props.focalPoint);
+    }
+  };
+
+  return (
+    <div className={classes.root}>
+      <h1 className={classes.title}>Basic with apply button</h1>
+      <p className={classes.description}>
+        This example is the same as the basic example, but it adds a button to apply the focal point changes to the
+        image.
```

Comment:
In this example you can play changing the focal point by dragging it, and once you are done you can click on the apply button and it will reflected on the example pictures

Reply:
Done